### DISCUSSION TITLE
Procedure as APIs. [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1906] [Discord Threads]
> 🙋‍♂️ **Speedymax02**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### As of now there are no plans to provide procedure as apis  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965884345715937300)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
